nextflow_schema.json

```diff
@@ -17,13 +17,13 @@
                     "format": "file-path",
                     "exists": true,
                     "schema": "assets/schema_input.json",
-                    "mimetype": "text/csv",
-                    "pattern": "^\\S+\\.csv$",
+                    "pattern": "^\\S+\\.(csv|tsv|json|yaml|yml)$",
```

Comment:
I think that we should stay restricted on `.csv` suffix as long as we only support the `text/csv` format.